source/_components/group.markdown

```diff
@@ -52,14 +53,24 @@ Configuration variables:
 Example of groups shown as views in the frontend.
 </p>
 
-If all entities are switches or lights they can be controlled as one with a switch at the top of the card. Grouped states should share the same type of states (ON/OFF or HOME/NOT_HOME).
+If all entities in a group are switches or lights then Home Assistant adds a switch at the top of the card that turns them all on/off at once.
+
+You can create views (tabs) that contain other groups.  
+Notice in the example below that in order to refer to the group "Living Room", you use `group.living_room` (lowercase and spaces replaced with underscores).
 
 ```yaml
-# Example configuration.yaml entry
-group:
```

Comment:
These configuration examples are here to show that there are three different ways to specify a group config.